framework/operator.go

```diff
@@ -2,8 +2,55 @@ package framework
 
 import (
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
 )
 
+// Config represents the configuration used to create a new operator framework.
+type Config struct {
+	// Dependencies.
+	Logger micrologger.Logger
+}
+
+// DefaultConfig provides a default configuration to create a new operator
+// framework by best effort.
+func DefaultConfig() Config {
+	var err error
+
+	var newLogger micrologger.Logger
+	{
+		config := micrologger.DefaultConfig()
+		newLogger, err = micrologger.New(config)
+		if err != nil {
+			panic(err)
+		}
+	}
```

Comment:
Lets talk about this separately. My main stand is that there is no consistency because the number of requirements is infinite. For instance I like to have a default logger being configured in tests which just discards all logs.